src/App.tsx

```diff
@@ -1,42 +1,44 @@
-import React from 'react';
+import React, { useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 import './App.scss';
+import { Tabs } from './components/Tabs/Tabs';
 
 export const tabs = [
   { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
   { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
   { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
 ];
 
+type Tab = {
+  id: string,
+  title: string,
+  content: string,
+};
+
 export const App: React.FC = () => {
+  const [selectedTabId, setSelectedTabId] = useState(tabs[0].id);
+
+  const currentTab = tabs?.find(tab => tab.id === selectedTabId) || tabs[0];
+
+  const { title } = currentTab;
+
+  const onTabSelected = (tab: Tab) => {
```

Comment:
```suggestion
  const handleTabSelected = (tab: Tab) => {
```
https://javascript.plainenglish.io/handy-naming-conventions-for-event-handler-functions-props-in-react-fc1cbb791364